Source/DafnyCompilers/CompilerCommon.dfy

```diff
@@ -74,7 +131,7 @@ module {:extern "DafnyInDafny.Common"} DafnyCompilerCommon {
         ProperMultiSubset | ProperMultiSuperset | MultisetDisjoint | InMultiset |
         NotInMultiset | MultisetUnion | MultisetIntersection | MultisetDifference |
         MultisetSelect // Separate node in DafnyAST.cs
-      datatype Maps =
+      datatype Maps = // TODO: MapSubtraction -> MapSubstraction? (with an "s"?)
```

Comment:
Not AFAIK, see https://english.stackexchange.com/questions/3640/is-substract-versus-subtract-a-proper-word